src/components/syncPlay/syncPlayManager.js

```diff
@@ -741,7 +741,7 @@ class SyncPlayManager {
 
         const playAtTime = this.lastCommand.When;
 
-        const currentPositionTicks = playbackManager.currentTime();
+        const currentPositionTicks = playbackManager.currentTime() * 10000;
```

Comment:
Better fix would be on the next line, since what it does is essentially compute the difference between server position and current position, then convert to ms.

So converting the `serverPositionTicks` to ms would allow us to not have to convert this, then convert the result to ms afterwards.